api.js

```diff
@@ -165,10 +167,14 @@ Api.prototype._prefixTitle = function (file) {
 	return prefix;
 };
 
-Api.prototype.run = function (files) {
+Api.prototype.run = function (files, forceHasExclusive) {
```

Comment:
Sure. Opted for `runOnlyExclusive` which is the same as the option passed to the runner.